manifests/init.pp

```diff
@@ -109,6 +109,15 @@
     notify  => Service[$freeradius::fr_service],
   }
 
+  # Add systemd unit to override default file on RHEL8 systems.
+  if ($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8') {
+    systemd::dropin_file { 'remove_bootstrap.conf':
+      ensure  => present,
+      unit    => 'radiusd.service', #@todo programmatically determine the service name
+      content => template('freeradius/systemd_dropin_rhel8.erb'),
+    }
```

Comment:
Let's rename this to be unique and make the filename not RHEL8 specific, as this impacts any EL8 and maybe other distros too, perhaps something like:

```suggestion
    systemd::dropin_file { 'freeradius remove bootstrap':
      filename => remove_bootstrap.conf,
      ensure   => present,
      unit     => 'radiusd.service', #@todo programmatically determine the service name
      content  => template('freeradius/systemd_dropin_remove_bootstrap.erb'),
    }
```

Reply:
Also - do you want to turn this todo in to an issue, I don't think we can solve that in this PR

Reply:
> @nward or @amateo are either of you guys on CentOS/RHEL? I guess they're highly likely to be the same as Rocky 8, but you never know... 😄

I actually do have some real world licensed RHEL8 boxes, but not running FreeRADIUS :( I am happy with this being tested on Rocky8 - I'm all over those RPMs and they're all the same in this regard!